Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java

```diff
@@ -89,7 +89,7 @@
      * This method is invoked if the user clicks on the add multi doc struc
      * submit btn command button.
      */
-    public void addMultiDocStrucSubmitBtnClick() {
+    public void addMultiDocStrucSubmit() {
```

Comment:
Is submit needed in the name?

Reply:
You are right, it's probably not needed!